ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Envelope.scala

```diff
@@ -29,7 +29,7 @@ object Envelope {
   private def enclose(
       kind: Proto.Envelope.MessageKind,
       bytes: ByteString,
-      compression: Boolean = true): ByteString =
+      compression: Boolean): ByteString =
```

Comment:
This default might be unused for now but I think it's meaningful.

Reply:
Is it meaningful enough to be silenced with a regex matching the error and a comment?

Reply:
114c1a2f3bbc4734e1dc103e89528a790894cfcc

Reply:
Maybe not worth a silencing annotation, unless a specific use for it is planned soon; in that case I'd also remove the comment. @miklos-da what do you think?

Reply:
I agree with @fabiotudone-da. I.e., let's remove the comment and set it to a default that we define as a constant explicitly everywhere we call this function. We'll definitely use it with different values once we get to storage optimization (i.e., compressing only large enough values).

Reply:
4666d04ce7a5dc19ce3b8a8b1a8e625aa07570e0